src/components/cards/InformationCard.tsx

```diff
@@ -40,9 +44,11 @@ const InformationCard = ({ data }: any) => {
         >
           <div className="relative">
             {data.Title}
-            {data.PARSED_MANUAL_TAGS.ACCESS === "Institutional Access" ? (
+            {dataAccess === "Institutional Access" ||
+            dataAccess === "Paid Service" ||
+            dataAccess === "Open Source" ? (
```

Comment:
I would suggest moving these above return and assign it to a variable with a name like `starred`